CONTRIBUTING.md

```diff
@@ -1,18 +1,61 @@
-## Contributing
+# AcousticBrainz contributing guidelines
```

Comment:
Would `contribution guidelines` be more correct? Otherwise to me it sounds like guidelines are contributing to something. :)